src/archivematicaCommon/lib/fileOperations.py

```diff
@@ -178,8 +178,12 @@ def updateFileLocation2(src, dst, unitPath, unitIdentifier, unitIdentifierType,
     f.currentlocation = dstDB
     f.save()
 
-def updateFileLocation(src, dst, eventType, eventDateTime, eventDetail, eventIdentifierUUID = uuid.uuid4().__str__(), fileUUID="None", sipUUID = None, transferUUID=None, eventOutcomeDetailNote = ""):
-    """If the file uuid is not provided, will use the sip uuid and old path to find the file uuid"""
+def updateFileLocation(src, dst, eventType, eventDateTime, eventDetail, eventIdentifierUUID = uuid.uuid4().__str__(), fileUUID="None", sipUUID = None, transferUUID=None, eventOutcomeDetailNote = "", createEvent=True):
+    """
+    If the file uuid is not provided, will use the sip uuid and old path to find the file uuid.
```

Comment:
Add a one-line summary to the beginning of the docstring?  "Updates file location in the database and optionally generates an Event.  NOTE does not actually move file. """ ?